features/facebookImport/src/model/analyses/utils/analysis-status.js

```diff
@@ -1,14 +1,12 @@
-export function createSuccessStatus(analysisClass) {
+export function createSuccessStatus() {
     return {
-        analysisClass,
         name: "SUCCESS",
```

Comment:
Sorry I've missed this before, but maybe we could define some kind of enum that constrains the success status we will obtain?

Reply:
This method will be removed in `feature/PROD4POD-1017-log-performance-importers`